proto/src/meta.rs

```diff
@@ -1,11 +1,12 @@
-// This file is generated by rust-protobuf 2.10.1. Do not edit
+// This file is generated by rust-protobuf 2.17.0. Do not edit
```

Comment:
How about leaving it untouched and switching to prost in next PR instead?

Reply:
Alright